### PR TITLE
enh(oidc): Improved OIDC chapter with new features

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 22.10.x (next)

### CODE REVIEW

**Comment by cg-tw** [comment]
waiting for FR translation of labels

File: versioned_docs/version-22.10/connect/openid.md [hunk 1]
```diff
@@ -39,7 +37,7 @@ Configure Identity Provider information:
 - **Scopes**: defines the scopes of the identity provider, for example `openid`. Separate scopes by spaces.
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/connect/openid.md:37]: Client ID and Client Secret are mandatory too

File: versioned_docs/version-22.10/connect/openid.md [hunk 2]
```diff
@@ -39,7 +37,7 @@ Configure Identity Provider information:
...
   > Depending on the identity provider, it is necessary to enter several scopes in order to retrieve the claim which will
   > identify users. This is indicated in the provider's configuration documentation.
-  - **Login claim value**: defines which of the variables returned by **Introspection Token Endpoint** or **User Information Endpoint**
+  - **Login attribute path**: defines which of the variables returned by **Introspection Token Endpoint** or **User Information Endpoint**
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/connect/openid.md:40]: image

Comment (mdebauchez) [reply versioned_docs/version-22.10/connect/openid.md:40]: I think the login attibute path should not be in the yellow part

File: versioned_docs/version-22.10/connect/openid.md [hunk 3]
```diff
@@ -58,21 +56,131 @@ You can also configure:
 > You can enable **Authentication debug** through the **Administration > Parameters > Debug** menu to understand
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/connect/openid.md:56]: Line 52 : The button name is 'Disable verify peer'

File: versioned_docs/version-22.10/connect/openid.md [hunk 4]
```diff
@@ -58,21 +56,131 @@ You can also configure:
...
+   ```
+
+   > Currently, only character string values can be used.
+
+### Step 4: Manage user creation
+
+<Tabs groupId="sync">
+<TabItem value="Users automatic management" label="Automatic management">
+
+If you turn on **Enable Auto import users**, users that log in to Centreon for the first time will be created in the Centreon configuration. (Turning the option on does not import automatically all users in your infrastructure.)
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/connect/openid.md:94]: The button name is 'Enable auto import'

File: versioned_docs/version-22.10/connect/openid.md [hunk 5]
```diff
@@ -58,21 +56,131 @@ You can also configure:
...
+
+</TabItem>
+</Tabs>
 
-### Step 5: Configure your Identity Provider (IdP)
+### Step 7: Configure your Identity Provider (IdP)
 
 Configure your IdP to add the Centreon application to use the OpenID Connect protocol to authenticate your users,
-And to authorize the following `redirect URI` to forward your connecter users to Centreon:
+And to authorize the following `redirect URI` to forward your connected users to Centreon:
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/connect/openid.md:183]: The letter A is uppercase

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md [hunk 6]
```diff
@@ -12,21 +12,19 @@ tout autre IdP compatible avec le flux d'autorisation via un code (Authorization
 
 ## Configurer l'authentification OpenID Connect
 
-L'authentification se paramètre à la page **Administration > Authentification > OpenId Connect Configuration** :
-
-![image](../assets/administration/oidc-configuration.png)
+L'authentification se paramètre à la page **Administration > Authentification > OpenId Connect Configuration**.
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md:15]: In french, the name of the page is Administration > Authentification > Configuration OpenId Connect

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md [hunk 7]
```diff
@@ -35,13 +33,14 @@ Renseignez les informations du fournisseur d'identité :
 - **URL de base** : définit l'URL de base du fournisseur d'identité pour les points d'entrée OpenId Connect (obligatoire).
 - **Point d'entrée d'autorisation** : définit le point d'entrée d'autorisation, par exemple `/authorize` (obligatoire).
 - **Point d'entrée de jeton** : définit le point d'entrée du jeton, par exemple `/token` (obligatoire).
-- **ID de client** : définit l'ID client.
+- **Identifiant client** : définit l'ID client.
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md:36]: Identifiant Client et Secret de client are mandatory too

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md [hunk 8]
```diff
@@ -51,30 +50,165 @@ Suivant votre fournisseur d'identité, définissez l'un ou l'autre des deux endp
 
 Vous pouvez également configurer :
 
-- **Utiliser l'authentification basique pour l'authentification du point d'entrée de jeton** : si cette option est activée, la méthode `Authorization: Basic` sera utilisée. Activez cette option si votre fournisseur d'identité le demande.
-- **Disable SSL verify peer** : permet de désactiver la validation des pairs SSL. Le certificat du fournisseur d'identité ne sera pas vérifié : cette option ne doit être utilisée qu'à des fins de test.
+- **Utiliser l'authentification basique pour l'authentification du point d'entrée de jeton** : si cette option est activée, la
+  méthode `Authorization: Basic` sera utilisée. Activez cette option si votre fournisseur d'identité le demande.
+- **Disable SSL verify peer** : permet de désactiver la validation des pairs SSL. Le certificat du fournisseur d'identité ne
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md:55]: The button name is 'Désactiver la vérification du pair'

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md [hunk 9]
```diff
@@ -51,30 +50,165 @@ Suivant votre fournisseur d'identité, définissez l'un ou l'autre des deux endp
...
+
+Si vous désactivez l'otion **Activer la gestion automatique**, vous devrez gérer manuellement les relations entre contacts et [groupes de contacts](../monitoring/basic-objects/contacts-groups.md#créer-un-groupe-de-contacts).
+
+</TabItem>
+</Tabs>
 
-### Step 5: Configurer le fournisseur d'identité
+### Étape 7 : Configurer le fournisseur d'identité
 
 Configurer votre fournisseur d'identité pour ajouter l'application Centreon à utiliser le protocole OpenID Connect pour
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md:210]: Configurez*

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Pull request merged**